_security/authentication-backends/jwt.md

```diff
@@ -0,0 +1,289 @@
+
+To use RS256, you need to configure only the (non-base64-encoded) public RSA key as `signing_key` in the JWT configuration:
+
+```yml
+jwt_auth_domain:
+  ...
+    config:
+      signing_key: |-
+        -----BEGIN PUBLIC KEY-----
+        MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQK...
+        -----END PUBLIC KEY-----
+      ...
+```
+
+The Security plugin automatically detects the algorithm (RSA/ECDSA), and if necessary you can break the key into multiple lines.
+
+
+### Bearer authentication for HTTP requests
+
+The most common way of transmitting a JWT in an HTTP request is to add it as an HTTP header with the bearer authentication schema:
+
+```
+Authorization: Bearer <JWT>
+```
+
+The default name of the header is `Authorization`. If required by your authentication server or proxy, you can also use a different HTTP header name using the `jwt_header` configuration key.
+
+As with HTTP basic authentication, you should use HTTPS instead of HTTP when transmitting JWTs in HTTP requests.
+
+
+### URL parameters for HTTP requests
+
+Although the most common way to transmit JWTs in HTTP requests is to use a header field, the Security plugin also supports parameters. Configure the name of the `GET` parameter using the following key:
+
+```yml
+    config:
+      signing_key: ...
+      jwt_url_parameter: "parameter_name"
+      subject_key: ...
+      roles_key: ...
+```
+
+As with HTTP basic authentication, you should use HTTPS instead of HTTP.
+
+
+### Validated registered claims
+
+The following registered claims are validated automatically:
+
+* "iat" (Issued At) Claim
+* "nbf" (Not Before) Claim
+* "exp" (Expiration Time) Claim
+
+
+### Supported formats and algorithms
+
+The Security plugin supports digitally signed, compact JWTs with all standard algorithms:
+
+```
+HS256: HMAC using SHA-256
+HS384: HMAC using SHA-384
+HS512: HMAC using SHA-512
+RS256: RSASSA-PKCS-v1_5 using SHA-256
+RS384: RSASSA-PKCS-v1_5 using SHA-384
+RS512: RSASSA-PKCS-v1_5 using SHA-512
+PS256: RSASSA-PSS using SHA-256 and MGF1 with SHA-256
+PS384: RSASSA-PSS using SHA-384 and MGF1 with SHA-384
+PS512: RSASSA-PSS using SHA-512 and MGF1 with SHA-512
+ES256: ECDSA using P-256 and SHA-256
+ES384: ECDSA using P-384 and SHA-384
+ES512: ECDSA using P-521 and SHA-512
+```
+
+
+## Using a JWKS endpoint to validate a JWT
+
+Validating the signature of the signed JWT is the last step in transmitting it from issuer to client. Rather than store the cryptographic key---either "shared secret" or "public key", depending on the algorithm used for signature validation---in the local `config.yml` file's `authc` section, you can specify a JSON Web Key Set (JWKS) endpoint to retrieve the key from where it's stored on the issuer's server. This method of validating the JWT can help streamline management of public keys and certificates.
```

Comment:
Validating the signature will be done by OpenSearch when a client sends the JWT with a REST Request. The signature will be verified on every request to authenticate the request. 

Otherwise this section looks good to me!

Reply:
@cwperks Reworded here. Thanks for your comments.